dev-docs/bidders/yieldlove

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: Yieldlove
+description: Yieldlove adapter
+biddercode: yieldlove
+pbjs: true
+gvl_id: 251
+gdpr_supported: true
```

Comment:
I would recommend adding

- safeframes_ok: true/false
- floors_supported: true/false
- userId: (list of supported vendors)

meta data if possible

Reply:
Thanks, I've added the data. We're currently not passing on any userIds, so I'll leave it out for now. Is that OK?

Reply:
Sure thing :)